// astro-opengraph-images/dist/presets/rauchg.js
import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
export async function rauchg({ title }) {
    return Promise.resolve(_jsxs("div", { style: {
            display: "flex",
            height: "100%",
            width: "100%",
            alignItems: "center",
            justifyContent: "center",
            letterSpacing: "-.02em",
            fontWeight: 700,
            background: "white",
        }, children: [_jsxs("div", { style: {
                    left: 42,
                    top: 42,
                    position: "absolute",
                    display: "flex",
                    alignItems: "center",
                }, children: [_jsx("span", { style: {
                            width: 24,
                            height: 24,
                            background: "black",
                        } }), _jsx("span", { style: {
                            marginLeft: 8,
                            fontSize: 20,
                        }, children: "your_site.com" })] }), _jsx("div", { style: {
                    display: "flex",
                    flexWrap: "wrap",
                    justifyContent: "center",
                    padding: "20px 50px",
                    margin: "0 42px",
                    fontSize: 40,
                    width: "auto",
                    maxWidth: 550,
                    textAlign: "center",
                    backgroundColor: "black",
                    color: "white",
                    lineHeight: 1.4,
                }, children: title })] }));
}
